ui/app/templates/components/job-editor/alert.hbs

```diff
@@ -0,0 +1,21 @@
+  {{#if @data.error}}
+    <Hds::Alert @type="inline" @color="critical" data-test-error={{@data.error.type}} as |A|>
+        <A.Title data-test-error-title>{{conditionally-capitalize @data.error.type true}}</A.Title>
```

Comment:
I don't feel great about handling this in JS, especially when the condition here is always true.

I'd suggest setting up a style (feel free to even make a new job-editor-alert.scss) that amounts to `.job-editor-alert .title { text-transform: capitalize; }`